app.js

```diff
@@ -0,0 +1,21 @@
+const express = require("express");
+const app = express();
+const { getTopics } = require("./controllers/topics-controllers");
+
+app.use(express.json());
```

Comment:
This should be removed as there is no need for it to be here right now. None of your endpoints are dealing with anything that has been sent in by the client on the request body.